myapp/templates/base.html

```diff
@@ -58,6 +58,9 @@
     </div>
     <div class="wrapper">
         <section class="container">
+            {% for message in messages %}

```

Comment:
nice, that's a new thing for me.

Reply:
Yeah, I discovered Django messages fairly recently. They are a great way to display quick status updates: https://www.webforefront.com/django/setupdjangomessages.html